app/Http/Controllers/Admin/JobPosterCrudController.php

```diff
@@ -67,6 +67,7 @@ public function setupListOperation()
             'type' => 'model_function',
             'function_name' => 'status'
         ]);
+
         $this->crud->addColumn([
```

Comment:
Should probably add another column to see which Jobs are "soft published" from the index view